frontend/src/App.tsx

```diff
@@ -576,8 +578,29 @@ export class App extends PureComponent<Props, State> {
     this.handleSessionStateChanged(initialize.sessionState)
   }
 
+  createThemeHash = (themeInput: CustomThemeConfig): string | null => {
+    if (!themeInput) {
+      return null
+    }
+
+    const themeInputEntries = Object.entries(themeInput)
+    themeInputEntries.sort()
+
+    const concatenatedTheme: string = themeInputEntries.reduce(
+      (themeStr: string, [themeOptName, themeOptVal]) =>
+        `${themeStr},${themeOptName}:${themeOptVal}`,
+      ""
+    )
```

Comment:
Do the delimiters matter? Because you can probably do: `themeInputEntries.join(":")` which basically inverses your delimter?

```javascript
// [ [ 'bar', 9 ], [ 'baz', 4 ], [ 'foo', 6 ] ]
Object.entries(baz).join(":")
// 'baz,4:foo,6:bar,9'
```

In any case, the reduce is clever and is readable enough.

Reply:
I actually added them just to make the stringified `themeInput` a bit more human-readable in case it ever had to be logged and inspected, so they're not important. Will switch to using `.join(":")`